lib/ts/index.ts

```diff
@@ -143,6 +143,15 @@ export default class AuthHttpRequest {
             userContext: getNormalisedUserContext(userContext)
         });
     };
+
+    static getAccessToken = async (input?: { userContext?: any }): Promise<string | undefined> => {
+        // This takes care of refreshing the access token if necessary.
+        await AuthHttpRequestFetch.recipeImpl.getAccessTokenPayloadSecurely({
+            userContext: getNormalisedUserContext(input === undefined ? undefined : input.userContext)
+        });
```

Comment:
if no session exists, this will throw an error - shouldn't we return `undefined` in this case from this function instead?

There are three cases:
1) session does not exist -> throw error?
2) session exists, cookie being used, so access token can't be read by frontend JS -> return undefined?
3) session exists, access token exists -> return token.

So what im saying is that case (1) and (2) should return `undefined`. Do you think that's a good idea? Or maybe case (1) should return `undefined`, but case (2) should throw an error? This way, if cookie transfer method is being used, user should not call this function.

Reply:
I think returning undefined in 1 and 2 should be fine.